cmd/duffle/export.go

```diff
@@ -92,20 +94,20 @@ func (ex *exportCmd) run() error {
 }
 
 func (ex *exportCmd) Export(bundlefile string, l loader.BundleLoader) error {
-	is, err := packager.NewImageStore(ex.thin, ex.ociLayout)
+	builder, err := builder2.NewBuilder(ex.thin, ex.ociLayout)
```

Comment:
nit: I'm not fan at all with the "one letter variable naming" we often see in go, but I think that here I'd prefer see something like this:
```go
b, err := builder.NewBuilder(...)
```
I think that package naming is more important than variable naming.

Reply:
Absolutely. This crept in due to refactoring without sufficient eye-balling afterwards. Fixed.